data/reusables/desktop/select-fork-behavior.md

```diff
@@ -0,0 +1,43 @@
+1. Click **Fork behavior**. 
+2. If you want the fork to make pull requests to the original upstream repository, select **To contribute to the parent repository**.
```

Comment:
Steps 2-3 should be one step with two sub-steps, like this:

```
1. Choose a fork behavior.
  - If you want...
  - If you want...
```

Although you could also just stop at "Choose a fork behavior" because the UI does a great job of explaining the two options.